src/test/osdc/object_cacher_stress.cc

```diff
@@ -47,7 +47,7 @@ class C_Count : public Context {
     : m_op(op), m_outstanding(outstanding) {}
   void finish(int r) override {
     m_op->done++;
-    assert(m_outstanding > 0);
+    assert(*m_outstanding > 0);
     *m_outstanding--;
```

Comment:
@chardan When I did `$git blame`, it showed up your name. So that's the reason I contacted you.

As I analyse the code once again now, it is maintaining a count of `outstanding_reads`. When the C_Count object is created it is incremented and the m_outstanding is initialized. Then upon complete(), it calls finish() and the value is decremented. The only usage of the outstanding_reads is for that assert and the assert is just a check before decrementing. So basically there is no need for maintaining such a count, unless it is used for something other than asserting.

@wjwithagen However, I'm working on this to see if we can keep this variable and just avoid the two warnings that we have. I will let you know my update in the evening IST, as the build takes too long  in my machine. :-)